agent/command/git.go

```diff
@@ -733,14 +733,28 @@ func (c *gitFetchProject) fetchModuleSource(ctx context.Context,
 		dir:      moduleBase,
 		method:   cloneMethod,
 	}
-	// Module's location takes precedence over the project-level clone
-	// method.
+	// If the module repo is using the deprecated ssh cloning method, extract the owner
+	// and repo from the string and construct a https cloning link manually.
+	// This is a temporary workaround which will be removed once users have switched over.
 	if strings.Contains(opts.location, "git@github.com:") {
-		opts.method = evergreen.CloneMethodLegacySSH
-	} else if opts.method == evergreen.CloneMethodOAuth {
+		logger.Task().Infof("ssh cloning is being deprecated. We are manually converting '%s'"+
```

Comment:
Should we add a mention of some EVG ticket to remove all this logic once the migration is done (if there is such a ticket)?

Reply:
Not yet. This is part of a larger effort so I don't think that's necessary right now.